testData/solutions/win32.json5

```diff
@@ -316,6 +316,219 @@
+                                                    "left": {
+                                                        "transform": {
+                                                            "type": "fluid.transforms.binaryOp",
+                                                            "leftPath": "http://registry\\.gpii\\.net/common/speechRate",
+                                                            "operator": "/",
+                                                            "right": 30
+                                                        }
+                                                    },
+                                                    "operator": "+",
+                                                    "right": 5
+                                                }
+                                            }
+                                        },
+                                        {
+                                            "output": 20
+                                        }
+                                    ]
+                                }
+                            }
+                        }
+                    },
+                    "SpeechPitch": {
+                        "transform": {
+                            "type": "fluid.transforms.linearScale",
+                            "inputPath": "http://registry\\.gpii\\.net/common/pitch",
+                            "factor": 20,
+                            "offset": 0
+                        }
+                    },
+                    "transform": [{
+                        "type": "fluid.transforms.arrayToSetMembership",
+                        "inputPath": "http://registry\\.gpii\\.net/common/trackingTTS",
+                        "outputPath": "",
+                        "presentValue": 1,
+                        "missingValue": 0,
+                        "options": {
+                            "focus": "CoupleNarratorCursorKeyboard",
+                            "caret": "FollowInsertion",
+                            "mouse": "InteractionMouse"
+                        }
+                    }],
+                    "EchoChars": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "false": 0,
+                            "true": 1,
+                            "conditionPath": "http://registry\\.gpii\\.net/common/keyEcho"
+                        }
+                    },
+                    "EchoWords": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "false": 0,
+                            "true": 1,
+                            "conditionPath": "http://registry\\.gpii\\.net/common/wordEcho"
+                        }
+                    }
+                }
```

Comment:
Unfortunately our transformation system is not yet clever enough to invert the speechRate transformation automatically, so you should provide an `"inverseCapabilitiesTransformations"` block here and define the inverse transformations... This should allow a user to step up to the system with NVDA only settings and automagically get a system with narrator in a usable settings range (as could be tested via an acceptance test using the screenreader_nvda preferences set, as is done in eg. orca)